custom_download_strategy.rb

```diff
@@ -103,6 +103,6 @@ def resolve_asset_id
 
   def fetch_release_metadata
     release_url = "https://api.github.com/repos/#{@owner}/#{@repo}/releases/tags/#{@tag}"
-    GitHub.open_rest(release_url)
+    GitHub::Api.open_rest(release_url)
```

Comment:
This should be `GitHub::API`; I got `Error: uninitialized constant GitHub::Api` running `bin/setup` in `performance-api`.

I can't push a branch for a PR though, @robjacoby do I need to be added to a group for the camp?

Reply:
@brentonannan good catch!! I've filed a PR https://github.com/cultureamp/homebrew-perform-cli/pull/3